molecule/deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       serviceAccountName: migration-operator
       containers:
       - name: operator
-        image: quay.io/konveyor/mig-operator-container:release-1.5.1
+        image: quay.io/konveyor/mig-operator-container:release-1.5.2
```

Comment:
This is not your fault, but I just noticed this shoudl be `quay.io/konveyor/mig-legacy-operator-container:release-1.5.2`, if you wouldn't mind fixing them. Otherwise I can open a separate PR.

Reply:
:+1: